cmd/tools/doctor/doctor.go

```diff
@@ -198,17 +198,20 @@ func printRedactedConfig(path string, contents []byte) string {
 }
 
 func sanitize(nodes []*yaml.Node) {
+	// Update this list when there are additional tokens to sanitize
 	sanitizeWords := []string{"username", "password", "grafana_api_token", "token",
 		"host", "addr"}
 	for i, node := range nodes {
 		if node == nil {
 			continue
 		}
 		if node.Kind == yaml.ScalarNode && node.ShortTag() == "!!str" {
-			// Update this list of conditionals if there are other tokens you want to sanitize
 			value := node.Value
 			for _, word := range sanitizeWords {
 				if value == word {
+					if nodes[i-1].Value == "auth_style" {
```

Comment:
what if auth_style is first param in poller?

F2240-127-26:
    auth_style: basic_auth
    datacenter: F2240-127-26